server/repository/src/migrations/v1_07_00/invoice_add_currency_fields.rs

```diff
@@ -9,8 +9,53 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
         r#"
         ALTER TABLE invoice ADD COLUMN currency_id TEXT REFERENCES currency(id);
         ALTER TABLE invoice ADD COLUMN currency_rate {DOUBLE};
-        ALTER TABLE invoice ADD COLUMN foreign_currency_total {DOUBLE};
+        ALTER TABLE invoice_line ADD COLUMN foreign_currency_price_before_tax {DOUBLE};
 
+        DROP VIEW IF EXISTS invoice_stats;
```

Comment:
True this is here! One nit though is I think it's better to remove `IF EXISTS`. It should 100% exist, and`IF EXISTS` covered up typos of the thing that meant to be getting dropped (or never existed to begin with) several times throughout merge feature changes.